src/app/hooks/useGa.ts

```diff
@@ -51,8 +52,7 @@ export const useGa = (): {
   };
 } => {
   const debugMode =
-    process.env.NEXT_PUBLIC_NODE_ENV !== "production" &&
-    process.env.NODE_ENV !== "test";
+    process.env.NODE_ENV !== "production" && process.env.NODE_ENV !== "test";
```

Comment:
Calling out that this is the only potentially behaviour-alternating change I made - AFAIK, `NEXT_PUBLIC_NODE_ENV` isn't set anywhere any more. I changed it to regular `NODE_ENV`, but if we also want to enable debug mode on stage, maybe it should be `APP_ENV`? /cc @rhelmer

Reply:
Hm, I think Next might automatically treat `NODE_ENV` as if it was `NEXT_PUBLIC_NODE_ENV` and compile it in... we should test. I think we might need to pass this as a prop (or alternatively you could have a server-side API that's called at runtime, maybe server actions make this easy enough?)

Reply:
Oh, that would be problematic indeed. But good point, it's probably best to keep the existing behaviour for now (even if it's "never matching" because it's undefined), then we can always revisit later: 9375c3544.